### DISCUSSION TITLE
寄付でMisskey.ioアカウントにバッジをつける運用を縮小する提案

### DISCUSSION THREAD

**Discussion by KisaragiEffective** [misskey-dev/misskey#14719] [General]
(※このdiscussionはコード面での問題ではないためissueには起票されていません)

現状、Misskey Project—もといsyuilo氏の口座へ通算1000円以上寄付することでmisskey.ioアカウントへバッジを付与するというプロセスが存在します。
しかし、misskey.ioを事実上統括する村上氏はmisskey.ioの連合制限をより強化する方向性を示しています (リンク: https://misskey.io/notes/9yve7su1y8k30g2m)
> 月49億リクエスト捌くのに今の金額じゃ厳しいから今後本格的に連合について制限かけていかないといけないかもなぁ

この方向性は「misskey.ioだけがMisskeyというソフトウェアを動かすサーバーである」という誤った推測を加速することになります。
そこで、misskey.ioが「公式サーバー」ではないことを示すために、上述した寄付特典について以下の3点を提案します。

1. 特典の新規付与停止。
2. 特典の完全な廃止。
3. misskey-hubの寄付特典説明ページにMisskey.ioとMisskey Projectは関係ないことをはっきり明記する。

以上、ご検討いただけると幸いです。

**Comment by syuilo** [comment]
> この方向性は「misskey.ioだけがMisskeyというソフトウェアを動かすサーバーである」という誤った推測を加速する

があまり理解できなかったのと、特典を停止すると、プロジェクトの収入がかなり消えてしまうことになるので、バッジをつける運用を縮小することは考えていません。

**Reply by KisaragiEffective** [reply]
代替の検討もせずにDiscussionが閉じられたことは遺憾に思いますが、それは置いておきます。
それはそれとして3については引き続きやるべきであると主張します。

**Comment by Sayamame-beans** [comment]
"Misskeyへの支援"において、ioでだけ付く特典があるというのに若干違和感があるような気は(言われてみれば)しますね…?

**Reply by samunohito** [reply]
「"Misskeyへの支援"をしてくれたら、misskey.ioとして特典を付与します」というだけな気も…？
（misskey-devが、ではなくmisskey.ioが主導で行っていること）
たとえば、io以外のサーバでも同様のキャンペーンを行えば条件は同じではありませんか？

（misskey-devの管理するリポジトリやドキュメントなどで、特定のサーバ上の特典について言及されていたら考慮する必要はあるかと考えますが…）

**Reply by KisaragiEffective** [reply]
もしmisskey.ioとしてやっているのであれば村上氏に連絡するのが順当なところでしょう。そこであえてsyuilo氏を窓口にする必要性が見えません。

**Reply by syuilo** [reply]
自分はmisskey.ioのメンバーかつmisskey-devメンバーなので、なにもおかしい点は無いと思います(他にこのポジションの人間はいません)

**Reply by VillSnow** [reply]
ioの他のサーバーでもmisskeyへの支援のバッジをつけられるように審査する仕組みを作ったら公平な気がします（が、その仕組みを作って運用するコストの問題がががが）

**Reply by Sayamame-beans** [reply]
> 「"Misskeyへの支援"をしてくれたら、misskey.ioとして特典を付与します」というだけな気も…？
（misskey-devが、ではなくmisskey.ioが主導で行っていること）

なるほど…納得です

> io以外のサーバでも同様のキャンペーンを行えば

そこはsyuiloさんがmisskey.ioのメンバーであるから出来ることではありそうですね (支援したことを証明出来ないため)

**Comment by syuilo** [comment]
検討をしていないと決めつけられるのは遺憾です

**Reply by KisaragiEffective** [reply]
私としては透明性のために公開することを推奨します。
